.github/workflows/unit_race.yml

```diff
@@ -14,7 +14,7 @@ env:
 jobs:
 
   build:
-    name: Unit Test (Race)
+    name: Unit Test (Non_Evalengine_Race)
```

Comment:
Along the lines of harshit's comment, it makes sense to revert this change as well.